src/main/java/com/lebediev/movieland/web/controller/utils/JsonConverter.java

```diff
@@ -2,57 +2,102 @@
+    public static String toJson(List<MovieDto> movieDtoList, JsonView jsonView) {
         try {
             if (JsonView.BASE.equals(jsonView)) {
-                return objectMapper.writerWithView(MovieViews.BaseMovie.class).writeValueAsString(movieDtoList);
+                return OBJECT_MAPPER.writerWithView(MovieViews.BaseMovie.class).writeValueAsString(movieDtoList);
             } else if (JsonView.EXTENDED.equals(jsonView)) {
-                return objectMapper.writerWithView(MovieViews.ExtendedMovie.class).writeValueAsString(movieDtoList);
+                return OBJECT_MAPPER.writerWithView(MovieViews.ExtendedMovie.class).writeValueAsString(movieDtoList);
             } else if (JsonView.REVIEW.equals(jsonView)) {
-                return objectMapper.writerWithView(MovieViews.MovieWithReview.class).writeValueAsString(movieDtoList);
+                return OBJECT_MAPPER.writerWithView(MovieViews.MovieWithReview.class).writeValueAsString(movieDtoList);
             }
             throw new IllegalArgumentException("Wrong parameters list.size(" + movieDtoList.size() + "), jsonView = " + jsonView);
         } catch (JsonProcessingException e) {
             throw new IllegalArgumentException(e);
         }
     }
 
-    public static String toJson(List <Genre> genreList) {
+    public static String toJson(List<?> list) {
         try {
-            return objectMapper.writeValueAsString(genreList);
+            return OBJECT_MAPPER.writeValueAsString(list);
         } catch (JsonProcessingException e) {
-            throw new IllegalArgumentException("Error in genre list");
+            throw new IllegalArgumentException("Error in list");
         }
     }
 
     public static String toJson(MovieDto movieDto, JsonView jsonView) {
         try {
             if (JsonView.BASE.equals(jsonView)) {
-                return objectMapper.writerWithView(MovieViews.BaseMovie.class).writeValueAsString(movieDto);
+                return OBJECT_MAPPER.writerWithView(MovieViews.BaseMovie.class).writeValueAsString(movieDto);
             } else if (JsonView.EXTENDED.equals(jsonView)) {
-                return objectMapper.writerWithView(MovieViews.ExtendedMovie.class).writeValueAsString(movieDto);
+                return OBJECT_MAPPER.writerWithView(MovieViews.ExtendedMovie.class).writeValueAsString(movieDto);
             } else if (JsonView.REVIEW.equals(jsonView)) {
-                return objectMapper.writerWithView(MovieViews.MovieWithReview.class).writeValueAsString(movieDto);
+                return OBJECT_MAPPER.writerWithView(MovieViews.MovieWithReview.class).writeValueAsString(movieDto);
             }
             throw new IllegalArgumentException("Wrong parameters (" + movieDto + "), jsonView = " + jsonView);
         } catch (JsonProcessingException e) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    public static String toJson(TokenDto tokenDto) {
```

Comment:
to many static overloaded methods. Maybe divide a job will be better?

Reply:
made some simplification